smtp_test.go

```diff
@@ -257,7 +257,7 @@ func testSendMailTimeout(t *testing.T, d *Dialer, want []string) {
 	doTestSendMail(t, d, want, true)
 }
 
-func doTestSMTPReset(t *testing.T, d *Dialer, want []string){
+func doTestSMTPReset(t *testing.T, d *Dialer, want []string) {
```

Comment:
If possible, could you add a test to ensure that `SendCustomFrom` is doing what you expect it to do? At the very least, we'll want to make sure that the `MAIL FROM` is what we expect and the `From` header is still the from the message.